src/test_helpers/linux.rs

```diff
@@ -142,16 +145,26 @@ impl WorkloadManager {
             local_ip: Some(ns.ip()),
             inpod_uds,
             proxy_mode,
+            testing_dedicated_mark: if proxy_mode == ProxyMode::Dedicated {
```

Comment:
Yeah - in ref to above comments our `inpod` and `dedicated_testing` marks are both 1337 - so maybe it should just be a single (overridable) "mark" config value, and we just use it differently according to ProxyMode